### PR TITLE
Keep zre_msg.h in src directory

### PR DESCRIPTION
**Pull request by stvales**
This is a bug in automatic code generation which places all headers in include by default

### CODE REVIEW

File: src/zre_msg.c [hunk 1]
```diff
@@ -35,7 +35,7 @@
 #endif
 
 #include "../include/zyre.h"
-#include "../include/zre_msg.h"
+#include "./zre_msg.h"
```
Comment (bluca) [inline src/zre_msg.c:38]: there's no need for ./, it's the default

Comment (stvales) [reply src/zre_msg.c:38]: Agreed, I was about to commit without it but I found out that the previous version had the './'

**Comment by bluca** [comment]
There are still errors, please see the CI results

**Comment by bluca** [comment]
src/zre_msg.c: In function 'zre_msg_encode':
src/zre_msg.c:1392:9: error: variable 'have_content' set but not used [-Werror=unused-but-set-variable]
     bool have_content = false;

**Comment by keent** [comment]
@stvales Yeah just delete it.

**Comment by stvales** [comment]
Making corrections right now. I take the opportunity to have tests pass on macos as well

**Comment by stvales** [comment]
Could you tell me if the pending errors come from my modifications ? It's unclear and might have been there before...

**Comment by bluca** [comment]
the failures in valgrind and asan are new

**Comment by stvales** [comment]
seems OK with valgrind and asan now

**Comment by stvales** [comment]
\o/

**Pull request merged**